IDEAS/Buildings/Components/Shading/Interfaces/ShadingProperties.mo

```diff
@@ -2,7 +2,10 @@ within IDEAS.Buildings.Components.Shading.Interfaces;
 record ShadingProperties
   "Default: no shading"
   extends Modelica.Icons.Record;
-  parameter Boolean controlled = false
+  parameter Boolean controlled = shaType==
+      IDEAS.Buildings.Components.Shading.Interfaces.ShadingType.Screen or shaType==
+      IDEAS.Buildings.Components.Shading.Interfaces.ShadingType.BoxAndScreen or shaType==
+      IDEAS.Buildings.Components.Shading.Interfaces.ShadingType.OverhangAndScreen
```

Comment:
More the ` or shaType==` at the end of every line to the start of the next line to improve readability.
E.g.
`controlled = `
`shaType == IDEAS.Buildings.Components.Shading.Interfaces.ShadingType.Screen`
`or shaType == IDEAS.Buildings.Components.Shading.Interfaces.ShadingType.BoxAndScreen`